onnxruntime/core/framework/op_kernel_context_internal.h

```diff
@@ -58,6 +58,7 @@ class OpKernelContextInternal : public OpKernelContext {
   const bool& GetTerminateFlag() const noexcept { return terminate_flag_; }
 
   const onnxruntime::concurrency::ThreadPool* GetOperatorThreadPool() const { return session_state_.GetThreadPool(); }
+  onnxruntime::concurrency::ThreadPool* GetOperatorThreadPool() { return session_state_.GetThreadPool(); }
 
```

Comment:
It may be better if we had a threadpool api where using the threadpool didn't require a non-const version, as returning a non-const threadpool instance from a const SessionState is a bit of a smell. Something to consider doing in the future.